lib/pretty-repl.js

```diff
@@ -42,7 +42,7 @@ function computeCommonPrefixLength(before, after) {
 // Count the number of Unicode codepoints in a string, i.e. [...str].length
 // without the intermediate Array instance.
 function characterCount(str) {
-  let i, c;
+  let i; let c;
```

Comment:
```suggestion
  let i;
  let c;
```

(although I’m surprised that the linter complained about this in the first place)